utils/scripts/ocm/ocm.py

```diff
@@ -1405,6 +1437,28 @@ def oc_login_as_admin(self):
     )
     dbaaspolicy_parser.set_defaults(func=ocm_obj.update_dbaaspolicy_config)
 
+    # Argument Parser to install RHODA for given version from Dev Catalog
+    install_rhoda_dev_parser = subparsers.add_parser(
+        "install_rhoda_dev",
+        help=("To Install RHODA from Develop Catalog Image"),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    required_install_rhoda_dev_parser = install_rhoda_dev_parser.add_argument_group("required arguments")
```

Comment:
Is this some kind of constructor for the class you created before this?

Reply:
This is part of argument python argparse library. More details can be found on this doc - https://docs.python.org/3/howto/argparse.html